src/test/macros_test.cpp

```diff
@@ -13,24 +13,34 @@ TEST(MacroRecordingTest, ClaimRecording) {
     EXPECT_EQ(recorder.isRecordingActive(), false);
     recorder.claimRecording();
     EXPECT_EQ(recorder.isRecordingActive(), false);
-    recorder.setState(MacroState::Armed);
+    recorder.startRecording();
+    EXPECT_EQ(ControlProxy(kMacroRecordingKey, "recording_status").get(), 1);
```

Comment:
replace these magic numbers with static_casts of the enum class

Reply:
Added a TODO :) https://github.com/mixxxdj/mixxx/pull/2873#discussion_r453794827